lib/confluence/set_ops.es6.js

```diff
@@ -295,24 +293,46 @@ foam.CLASS({
 
   methods: [
     function SET_MINUS(dao, sink) {
+      if (Array.isArray(dao))
+        return this.cascadeSink_(this.IntersectDecider, dao, sink);
```

Comment:
Also this PR should have some tests added because the existing tests weren't sufficient to catch this problem.